libs/libpahomqtt/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libpahomqtt
+PKG_VERSION:=1.3.9
+PKG_RELEASE:=1
+PKG_SOURCE_PROTO:=git
+
+PKG_SOURCE_URL:=git://github.com/eclipse/paho.mqtt.c.git
+PKG_SOURCE_VERSION:=3b7ae6348bc917d42c04efa962e4868c09bbde9f
+PKG_MIRROR_HASH:=
+
+PKG_MAINTAINER:=Nian Bohung <n0404.n0404@gmail.com>
+PKG_LICENSE:=Eclipse Public License 2.0
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/libpahomqtt
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Eclipse paho mqtt c library
+  URL:=git://github.com/eclipse/paho.mqtt.c.git
+endef
+
+CMAKE_OPTIONS:=-DPAHO_BUILD_STATIC=TRUE -DPAHO_WITH_SSL=FALSE -DPAHO_HIGH_PERFORMANCE=TRUE
```

Comment:
Why you need static build? Why did you disabled SSL?

My proposal:

```make
CMAKE_OPTIONS += \
	-DPAHO_WITH_SSL=TRUE \
	-DPAHO_HIGH_PERFORMANCE=TRUE
```

Reply:
I have already modified and enabled SSL.
Because I will compile source code on OpenWrt over the static library, enable the static.

Reply:
is there any _reason_ that you need to make static builds though?  is that something that everyone needs to be building?

Reply:
I have already removed the option "PAHO_BUILD_STATIC".
Thank you.